samples/v1.2/Scenarios/Agenda.json

```diff
@@ -137,7 +140,8 @@
 										{
 											"type": "Image",
 											"horizontalAlignment": "Left",
-											"url": "https://messagecardplayground.azurewebsites.net/assets/Conflict.png"
+											"url": "https://messagecardplayground.azurewebsites.net/assets/Conflict.png",
+											"altText": "Conflict"
```

Comment:
Maybe slightly more descriptive? "Source Conflict icon" or something?  Btw I wonder if these URL's need to be updated at all? The raw URL takes me to the messagecardsplayground landing page not hte image itself ?